scwx-qt/source/scwx/qt/ui/edit_marker_dialog.cpp

```diff
@@ -0,0 +1,311 @@
+#include "edit_marker_dialog.hpp"
+#include "ui_edit_marker_dialog.h"
+
+#include <scwx/qt/manager/marker_manager.hpp>
+#include <scwx/qt/types/marker_types.hpp>
+#include <scwx/qt/util/color.hpp>
+#include <scwx/qt/util/q_color_modulate.hpp>
+#include <scwx/util/logger.hpp>
+
+#include <string>
+
+#include <QObject>
+#include <QString>
+#include <QIcon>
+#include <QPixmap>
+#include <QColorDialog>
+#include <QPushButton>
+#include <QFileDialog>
+
+namespace scwx
+{
```

Comment:
If we want to start moving to combined namespaces, that is fine, but will likely mean inconstancy for the time being. Otherwise this should be disabled.